.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,6 @@
+A checklist to help your pull request get merged faster:
+- [ ] Summarize your changes
+- [ ] Data: link to supporting info (examples: docs, changelogs, source control, bug trackers, tests)
```

Comment:
Not sure if it is crystal clear what we mean with "supporting info" here. Maybe:

`- [ ] Provide resources that help verifying support information (links to the browser's docs, changelogs, source control, bug trackers, tests)`

Reply:
Good idea. Pushed another version of this!